tools/ci_build/github/linux/docker/scripts/install_centos.sh

```diff
@@ -7,16 +7,33 @@ os_major_version=$(cat /etc/redhat-release | tr -dc '0-9.'|cut -d \. -f1)
 if ! rpm -q --quiet epel-release ; then
   yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-$os_major_version.noarch.rpm
 fi
+
+echo "installing for os major version : $os_major_version"
 if [ "$os_major_version" == "5" ]; then
- #Be careful, don't pull gcc into the base system, because we already have one in /opt/rh/devtoolset-2/root/usr/bin
- yum install -y redhat-lsb expat-devel libcurl-devel tar unzip curl zlib-devel make  python2-devel icu  rsync bzip2 git bzip2-devel
+  #Be careful, don't pull gcc into the base system, because we already have one in /opt/rh/devtoolset-2/root/usr/bin
+  yum install -y redhat-lsb expat-devel libcurl-devel tar unzip curl zlib-devel make  python2-devel icu  rsync bzip2 git bzip2-devel
+elif [ "$os_major_version" == "6" ]; then
```

Comment:
```bash
elif [ "$os_major_version" == "6" ] && [ ! -d "/opt/python/cp35-cp35m" ]
```
The manylinux2010 docker image is also based on CentOS 6, it already has python installed under /opt. And we probably don't want to install another python.

Reply:
Thanks!

---
In reply to: [336302009](https://github.com/microsoft/onnxruntime/pull/2159#discussion_r336302009) [](ancestors = 336302009)